templates/layouts/Default:-petition-and-scroll-to-share-greenpeace.liquid

```diff
@@ -141,7 +158,7 @@
     });
 
     var interval = setInterval(function() {
-      if(SharePop && SharePop.FB){
+      if(window.SharePop && window.SharePop.FB){
```

Comment:
Are we deleting this?

Reply:
This is to handle the redirect after the share is made, that's why I left it there. I added the `window` prefix since we're getting errors on production when SharePop is undefined.